### PR TITLE
Add bitvector32 and fix constant length return

### PR DESCRIPTION
**Pull request by terencechain**
Required for Altair sync committee `minimal` size.

Also fixed BitVector64 and 256 to return constant length 64 and 256, instead of using math

### CODE REVIEW

File: bitvector32.go [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+var _ = Bitfield(Bitvector32{})
+
+// Bitvector32 is a bitfield with a fixed defined size of 32. There is no length bit
+// present in the underlying byte array.
+type Bitvector32 []byte
+
+// NewBitvector32 creates a new bitvector of size 32.
+func NewBitvector32() Bitvector32 {
+	byteArray := [8]byte{}
```
Comment (prestonvanloon) [inline bitvector32.go:15]: Why 8 byte array? Should it be 4 bytes?

**Review by prestonvanloon — CHANGES_REQUESTED** [review]

File: bitvector32.go (hunk 1 above)
Comment (prestonvanloon) [reply bitvector32.go:15]: 4 * 8 = 32

File: bitvector32.go [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+		b[idx/8] |= bit
+	} else {
+		b[idx/8] &^= bit
+	}
+}
+
+// Len returns the number of bits in the bitvector.
+func (b Bitvector32) Len() uint64 {
+	return uint64(len(b) * 8)
+}
```
Comment (prestonvanloon) [inline bitvector32.go:51]: Len returns the number of bits in the bitvector? That doesn't seem to match the implementation. Wouldn't it always be 32?

**Comment by terencechain** [comment]
Please take another look @prestonvanloon

**Review by prestonvanloon — APPROVED** [review]
Looks good to me. We discussed offline that the .Bytes(), .BitIndices(), and .Count() methods do not behave appropriately when the underlying slice is improperly set to a slice larger than .Len()/8. I'll fix this in a follow up PR for all of the bitvector implementations

**Pull request merged**